components/resources/ai_chat_ui_strings.grdp

```diff
@@ -288,6 +288,18 @@
   <message name="IDS_CHAT_UI_MENU_DELETE_CONVERSATION" desc="Menu entry for removing a conversation">
     Delete conversation
   </message>
+  <message name="IDS_CHAT_UI_REASONING_LABEL" desc="A label shown while AI Assistant is reasoning the conversation response">
+    Reasoning...
```

Comment:
Out of interest, why did we go with `Reasoning...` over `Thinking...`?

What are you doing?
Reasoning.

sounds a bit odd 

whereas

What are you doing?
Thinking.

sounds slightly more natural to me :laughing: No strong feelings :smile: